src/board/nap/nap_ext_event.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
+ * Contact: Henry Hallam <henry@swift-nav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <libswiftnav/logging.h>
+#include <libsbp/ext_events.h>
+
+#include "./nap_common.h"
+#include "../../settings.h"
+#include "../../timing.h"
+#include "../../sbp.h"
+
+/** \defgroup ext_events External Events
+ * Capture accurate timestamps of external pin events
+ * \{ */
+
+typedef enum {
+  NONE,
```

Comment:
If you rely on the numerical value of these then you should explicitly define them